utils/import-help-article.py

```diff
@@ -0,0 +1,59 @@
+import os
+import requests
+import sys
+from urllib.parse import urlparse
+
+import pypandoc
+from bs4 import BeautifulSoup
+
+if len(sys.argv) < 2:
+    sys.exit("Give the URL with the command")
+
+page_url = sys.argv[1]
```

Comment:
Not requesting any changes here, so this is more of a FYI. I find that using [argparse](https://docs.python.org/3/library/argparse.html) makes building Python CLI tools super easy and more human friendly. Just wanted to share my thought here :)

Reply:
:+1: Thanks for the tip! Take a look at the updated version if you have a minute